config.md

```diff
@@ -0,0 +1,57 @@
+established for the deployment.   Environments which are designated as `mesos.hdfs.native-hadoop-binaries` == true in the `mesos-site.xml`,
+there is no refresh of the `hdfs-site.xml` file.
+
+The mesos-site.xml file is used to configure the hdfs-mesos framework.  We are working to deprecated this file.  This general establishes 
+values for the scheduler and in many cases these are passed to the executors.  Although the configuration of the scheduler can be handled 
+via XML configuration, we encourage the use of system environment variables for this purpose.
+
+## Configuration Options
+
+* mesos.hdfs.framework.name -  Used to define the framework name.  This allows for 1) multi-deployments of hdfs and 2) has an impact on the dns name of the service.  The default is "hdfs".
+* mesos.hdfs.user - Used to define the user to use for the scheduler and executor processes.  The default is root.
+* mesos.hdfs.role - Used to determine the mesos role this framework will use.  The default is "*".
+* mesos.hdfs.mesosdns - true if mesos-dns is used.  The default is false.
+* mesos.hdfs.mesosdns.domain - When using mesos-dns, this value is the suffix used by mesos-dns.  The default is "mesos".
+* mesos.native.library - The location of libmesos library.  The default is "/usr/local/lib/libmesos.so"
+* mesos.hdfs.journalnode.count - The number of journal nodes the scheduler will maintain. The default is 3.
+* mesos.hdfs.data.dir -  The location to store data on the slaves.  The default is "/var/lib/hdfs/data".
+* mesos.hdfs.domain.socket.dir - The location used for a local socket used by the data nodes.  The default is "/var/run/hadoop-hdfs".
+* mesos.hdfs.backup.dir - The location to replicated data to as a backup.  The default is blank.
+* mesos.hdfs.native-hadoop-binaries -  This is true if hdfs is pre-installed on the slaves.  This will result in no distribution of binaries to the slaves.  It will also mean that no xml configure refresh will be provided to the slaves.  The default is false.
+* mesos.hdfs.framework.mnt.path - If native-hadoop-binaries == false, this is the location a symlink will be provided to execute hdfs commands on the slave.  The default is "/opt/mesosphere"
+* mesos.hdfs.state.zk - The zookeeper that the scheduler will use to store state.  The default is "localhost:2181"
+* mesos.master.uri - The zookeeper or mesos-master url that will be used to discover the mesos-master for scheduler registration. The default is "localhost:2181"
+* mesos.hdfs.zkfc.ha.zookeeper.quorum - The zookeeper that HDFS (not the framework) will use for HA mode.  The default is "localhost:2181"
+
+There are additional configurations for executor jvm and resource management of the nodes.
+
+## System Environment Variables
+
+All of configuration flags previous defined can be override with system environment variables.  The format to use to over a variable is to
```

Comment:
"All of the configuration flags previously defined can be overridden by system environment variables."